### ISSUE TITLE
Implement navigation view components

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#32]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To have a uniform mechanism of rendering navigation menus:

* A top/bottom navigation view component
* A culture switch view component used by the former one internally

### Why

Navigation and culture switching is ubiquituous, hence view components are a good fit.

### Issue expands upon

This issue expands upon another issue(s): #19 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
